schemas/models/structured_model.json

```diff
@@ -2,34 +2,48 @@
   "$schema": "http://json-schema.org/draft-07/schema",
   "$id": "https://raw.githubusercontent.com/bettyblocks/json-schema/acceptance/schemas/models/structured_model.json",
   "title": "Question",
-  "description": "An object representing a single question in a questionnaire, including the prompt to be displayed to the user and any necessary data to properly evaluate their response.",
-  "type": "object",
+  "description": "An Json Schema to validate client json schema in regards to structured models",
```

Comment:
Its called schema models. And its not used for schema models but for the Object property

Reply:
I've changed the description to:

"A Json Schema to validate client Json Schema in regards to the object property"